openedx/features/caliper_tracking/tests/expected/edx.team.learner_removed_remove_by_admin.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Removed",
+  "actor": {
+    "id": "http://localhost:18000/u/honor",
+    "name": "honor",
+    "type": "Person"
+  },
+  "eventTime": "2019-01-01T07:05:11.198Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36",
+      "course_id": "course-v1:edX+DemoX+Demo_Course",
+      "event_source": "server",
+      "event_type": "edx.team.learner_removed",
+      "host": "e5d2c598af1a",
+      "ip": "172.18.0.1",
+      "org_id": "edX",
+      "page": null,
+      "path": "/api/team/v0/team_membership/check-3429fc5983a84c8c8366a4477b03d91c,honor",
+      "session": "dff5c86a2aa1ebe2ea76bfd226e098a3",
+      "user_id": 6
+    }
+  },
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "object": {
+    "extensions": {
+      "remove_method": "remove_by_admin"
+    },
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/#teams/Topic1ID/check-3429fc5983a84c8c8366a4477b03d91c",
+    "member": {
+      "extensions": {
+        "user_id": 2
+      },
+      "id": "http://localhost:18000/u/honor",
```

Comment:
The person in object should be different from the actor. You cannot use the same user in both cases because admin is removing someone else.

Reply:
Done